geometry_msgs/PoseStamped object_pose
---
moveit_msgs/Grasp[] grasps
